src/audit-logs/audit-logs.spec.ts

```diff
@@ -24,29 +27,23 @@ const event: CreateAuditLogEventOptions = {
     },
   ],
   context: {
-    location: ' 192.0.0.8',
-    user_agent: 'Firefox',
+    location: '192.0.0.8',
+    userAgent: 'Firefox',
   },
   metadata: {
     successful: true,
   },
 };
 
-const serializeEventOptions = (options: CreateAuditLogEventOptions) => ({
-  ...options,
-  occurred_at: options.occurred_at.toISOString(),
-});
-
 describe('AuditLogs', () => {
   describe('createEvent', () => {
     describe('with an idempotency key', () => {
       it('includes an idempotency key with request', async () => {
-        mock
-          .onPost('/audit_logs/events', {
-            event: serializeEventOptions(event),
-            organization_id: 'org_123',
-          })
-          .replyOnce(201, { success: true });
+        const workosSpy = jest.spyOn(WorkOS.prototype, 'post');
```

Comment:
Encountered a really gnarly bug in `axios-mock-adapter` where the deep equality failed if a value was `undefined`. This meant that many of my serialized mocks were no longer being caught. I'm going to work on moving away from `axios-mock-adapter` to just Jest Mock with each of these PRs.